frontend/src/main/pages/PersonalSchedules/PersonalSchedulesDetailPage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useParams } from "react-router-dom";
+//import { useBackend } from 'main/utils/useBackend'
+
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+//import { useCurrentUser } from 'main/utils/currentUser';
+//import { useBackend } from 'main/utils/useBackend';
+//import PersonalSchedulesDetailTable from 'main/components/PersonalSchedulesDetailTable';
+// Remember to ask -1 team for components for DetailTable and edit based on that
+//import { useCurrentUser } from 'main/utils/currentUser'
+
+export default function PersonalSchedulesDetailPage() {
+
+    let { id } = useParams();
+
+    // const currentUser = useCurrentUser();
```

Comment:
Can we remove this commented out code?

It will be cleaner to just add back the correct code; a lot of this is specific to the edit page from which this was copied.   The best time to remove it is now.